packages/core/src/components/slider/handleProps.tsx

```diff
@@ -37,9 +37,6 @@ export interface IHandleProps extends IProps {
     /** Numeric value of this handle. */
     value: number;
 
-    /** Whether this handle is non-interactive. */
-    disabled?: boolean;
```

Comment:
add this to `IHandleInternalProps` then? pretty sure it is respected in `<Handle>`

Reply:
Yeah, fixed.

Reply:
tho it would be easy enough to support this prop, like you did with `onChange/onRelease`